modules/snowflake_connector_config/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  connector_name = "${var.environment_name}-snowflake-connector"
```

Comment:
may not be ideal yet

Reply:
Since we'll probably have only two connectors, shorter is probably better